packages/@poupe-tailwindcss/src/flat/plugin.ts

```
@@ -129,6 +136,35 @@ export function processParam(options: Partial<ThemeOptions>, debug: boolean, key
+        default:
+          if (!validThemePrefix(v)) {
+            // invalid value
+            return false;
+          }
+
+          options.surfacePrefix = v;
+      }
+
+      debugLog(debug, 'surface-prefix', options.surfacePrefix);
+      return true;
+    };
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)